netebpfext/net_ebpf_ext.h

```diff
@@ -143,6 +143,22 @@ typedef struct _net_ebpf_extension_wfp_filter_context
     HANDLE wfp_engine_handle;  ///< WFP engine handle.
 } net_ebpf_extension_wfp_filter_context_t;
 
+#ifdef _DEBUG
+#define CLEAN_UP_FILTER_CONTEXT(filter_context)                                \
+    if ((filter_context) != NULL) {                                            \
+        if ((filter_context)->filter_ids != NULL) {                            \
+            ExFreePool((filter_context)->filter_ids);                          \
+        }                                                                      \
+        if ((filter_context)->client_contexts != NULL) {                       \
+            ExFreePool((filter_context)->client_contexts);                     \
+        }                                                                      \
+        if ((filter_context)->wfp_engine_handle != NULL) {                     \
+            FwpmEngineClose((filter_context)->wfp_engine_handle);              \
+        }                                                                      \
+        net_ebpf_ext_remove_filter_context_from_zombie_list((filter_context)); \
+        ExFreePool((filter_context));                                          \
```

Comment:
I don't think you need the double parens in these lines.
```suggestion
        net_ebpf_ext_remove_filter_context_from_zombie_list(filter_context);   \
        ExFreePool(filter_context);                                            \
```

Reply:
I have generally followed the practice of using parentheses around the macro parameter to ensure that there are no accidental issues with any sort of operator precedence when the macro is expanded. I don't think it's strictly necessary in this case, but I prefer to keep it in.

This is also staying consistent with the original author's usage in this macro.